tests/tck/features/match/SeekByEdge.feature

```diff
@@ -0,0 +1,937 @@
+Feature: Match seek by edge
+  Examples:
+    | space_name  |
+    | nba         |
+    | nba_int_vid |
+
+  Background: Prepare space
+    Given a graph with space named "<space_name>"
+
+  Scenario: seek by edge index
+    When executing query:
+      """
+      MATCH (player)-[:serve]->(team)
+      RETURN player.name, team.name
+      """
+    Then the result should be, in any order:
```

Comment:
Can you add filter or limit operators to make the number of query results less for easy verification?

Reply:
NCK, mark resolved.